src/integer_mod_q/modulus_polynomial_ring_zq/get.rs

```diff
@@ -163,6 +163,28 @@ impl ModulusPolynomialRingZq {
     pub fn get_degree(&self) -> i64 {
         unsafe { fq_ctx_degree(self.get_fq_ctx_struct()) }
     }
+
+    /// Returns a representative polynomial of the [`ModulusPolynomialRingZq`] element.
+    ///
+    /// The representation of the coefficients is in the range `[0, modulus)`.
+    ///
+    /// # Examples
+    /// ```
+    /// use qfall_math::integer::PolyOverZ;
+    /// use qfall_math::integer_mod_q::ModulusPolynomialRingZq;
+    /// use std::str::FromStr;
+    ///
+    /// let modulus = ModulusPolynomialRingZq::from_str("4  -3 0 31 1 mod 17").unwrap();
+    ///
+    /// let poly_z = modulus.get_representative_0_modulus();
+    ///
+    /// let cmp_poly = PolyOverZ::from_str("4  14 0 14 1").unwrap();
+    /// assert_eq!(cmp_poly, poly_z);
+    /// ```
+    pub fn get_representative_0_modulus(&self) -> PolyOverZ {
+        let poly_zq = PolyOverZq::from(self);
+        poly_zq.get_representative_0_modulus()
+    }
```

Comment:
what do you think about the name:
- ``get_representative_least_positive_residue``

This seems to be more of a standardised name and at first I was also somewhat confused what was meant by the current name. What do you think @jnsiemer ?

Reply:
One could also rename the other function to ``get_representative_least_absolute_residue`` to be consistent if you like the suggestion. Although I do not find the other functions name to be as misleading